### PR TITLE
Stepper fix added

### PR DESCRIPTION
**Pull request by yerramshilpa**

### CODE REVIEW

File: src/app/data-entry-multi-form/data-entry-multi-form.component.ts [hunk 1]
```diff
@@ -66,39 +70,50 @@ export class DataEntryMultiFormComponent implements OnInit {
...
+        }
+      }
+    };
+    flatten(results);
+    return flat;
+  }
+
+  onPanelChange(id: string) {
+    this._currentStepIndex = this._dataEntryStepsDef.findIndex(item => item.id === id);
+    this.getField(id, this.dataEntryForm.steps)
```
Comment (Bibek-Regmi) [inline src/app/data-entry-multi-form/data-entry-multi-form.component.ts:116]: is this getField needed anymore?

File: src/app/data-entry-multi-form/data-entry-multi-form.component.ts [hunk 2]
```diff
@@ -66,39 +70,50 @@ export class DataEntryMultiFormComponent implements OnInit {
...
+  public getFlatElements() {
+    const results = this.dataEntryForm.steps;
+    const flat: DataEntrySteps[] = [];
+    const flatten = (array: any) => {
+      for (let i in array) {
+        const item = array[i];
+        flat.push({
+          id: item.id,
+          text: item.label
+        });
```
Comment (Bibek-Regmi) [inline src/app/data-entry-multi-form/data-entry-multi-form.component.ts:101]: Do we need to truncate data? Why not just push the item itself in?

File: src/app/data-entry-multi-form/data-entry-multi-form.component.ts (hunk 1 above)
Comment (yerramshilpa) [reply src/app/data-entry-multi-form/data-entry-multi-form.component.ts:116]: Yes, it's needed to get the current field

File: src/app/data-entry-multi-form/data-entry-multi-form.component.ts (hunk 2 above)
Comment (yerramshilpa) [reply src/app/data-entry-multi-form/data-entry-multi-form.component.ts:101]: I need only label and text for the stepper, don't need all of the object

**Pull request merged**